src/index.test.ts

```diff
@@ -1,9 +1,7 @@
-import greeter from '.';
+import * as mainModule from '.';
```

Comment:
Instead of this file, what do you think about requiring that all other test files import `.` instead of their respective files? For instance, `src/json.test.ts` could import `.` instead of importing `./json`.